client/src/component/Navigation/Bar.jsx

```diff
@@ -7,10 +7,9 @@ import history from '../../Routing/history';
 const Bar = () => {
     return (
         <div>
-            <button className={styleme.button1} onClick={() => history.push('/Review')}>Review</button>
-            <button className={`${styleme.button2}`} onClick={() => history.push('/Profile')}>Profile</button>
-            <button className={`${styleme.button3} ${styleme.tablink}`} onClick={() => history.push('/Nearby')}>Nearby</button>
-            <button className={`${styleme.button4}`}>logout</button>
+            <button className={styleme.button1} onClick={() => history.push('/Review')}><i className="fa fa-star" style={{'fontSize':'2.5rem'}} value="5"></i> </button>
```

Comment:
Keep the '</button>' spacing the same throughout all </button> tags. This first one has a space towards the end the others do not.